vignettes/test-small-cell-size.Rmd

```diff
@@ -40,5 +40,5 @@ Call your function that handles small cell sizes
 ```{r echo=FALSE}
 # TODO Add the code here
 library("SmallCells")
-TableOne <- small_size_cells(TableOne)
-```
\ No newline at end of file
+TableOne <- check_for_small_size_cells(TableOne)
```

Comment:
I'd probably call it `checkSmallCells`.

Reply:
Implemented and other function as well Standardized in pascal case